server/data/mappers/educationAndTrainingMapper.ts

```diff
@@ -0,0 +1,105 @@
+import moment from 'moment'
+import type { AchievedQualification, InductionResponse, InPrisonTrainingInterest } from 'educationAndWorkPlanApiClient'
+import type {
+  EducationAndTraining,
+  EducationAndTrainingData,
+  EducationAndTrainingLongQuestionSet,
+  EducationAndTrainingShortQuestionSet,
+} from 'viewModels'
+import toInductionQuestionSet from './inductionQuestionSetMapper'
+import educationalQualificationComparator from './educationalQualificationComparator'
+import enumComparator from './enumComparator'
+
+/**
```

Comment:
The mappers are essentially the same as what we originally had with the original mappers that map from `CiagInduction` - the difference is that the method arg is an `InductionResource` instead of `CiagInduction`, and the implementation detail is different in respect of the fields that it maps from.
The output and how it is built is essentially the same though 👍